src/components/shared/Entity/Create/index.tsx

```diff
@@ -161,78 +141,67 @@ function EntityCreate({
             ? hasLength(imageTag.id)
             : hasLength(imageTag.id) && !entityNameChanged && persistedDraftId;
 
-    if (showConnectorTiles === null) return null;
     return (
         <>
             <Box sx={{ maxHeight: 200, overflowY: 'auto', mb: 2 }}>
                 {errorSummary}
             </Box>
 
-            <Collapse in={showConnectorTiles} unmountOnExit>
-                <Typography sx={{ mb: 2 }}>
-                    <FormattedMessage id="entityCreate.instructions" />
-                </Typography>
-
-                <ConnectorTiles protocolPreset={entityType} replaceOnNavigate />
-            </Collapse>
-
-            <Collapse in={!showConnectorTiles} unmountOnExit>
-                {connectorTagsError ? (
-                    <Error error={connectorTagsError} />
-                ) : (
-                    <>
-                        {toolbar}
-
-                        <Collapse in={formSubmitError !== null} unmountOnExit>
-                            {formSubmitError ? (
-                                <EntityError
-                                    title={formSubmitError.title}
-                                    error={formSubmitError.error}
-                                    logToken={logToken}
-                                    draftId={draftId}
-                                />
-                            ) : null}
-                        </Collapse>
-
-                        {!isValidating && connectorTags.length === 0 ? (
-                            <AlertBox severity="warning">
-                                <FormattedMessage
-                                    id={`${messagePrefix}.missingConnectors`}
-                                />
-                            </AlertBox>
-                        ) : connectorTags.length > 0 ? (
-                            <ErrorBoundryWrapper>
-                                <DetailsForm
-                                    connectorTags={connectorTags}
-                                    accessGrants={combinedGrants}
-                                    entityType={entityType}
-                                />
-                            </ErrorBoundryWrapper>
+            {connectorTagsError ? (
```

Comment:
Assuming this chunk is like the ones above where changing the indentation makes the compare rough. Everything looks identical but did not deeply review each line.